openfl-tutorials/interactive_api/Flax_CNN_CIFAR/README.md

```diff
@@ -0,0 +1,87 @@
+# Federated FLAX CIFAR-10 CNN Tutorial
+
+### 1. About dataset
+
+The CIFAR-10 dataset consists of 60000 32x32 colour images in 10 classes, with 6000 images per class. There are 50000 training images and 10000 test images.
+
+Define the below param in envoy.yaml config to shard the dataset across participants/envoy.
+- rank_worldsize
+
+### 2. About model
+
+A simple multi-layer CNN is used with XLA compiled and Auto-grad based parameter updates.
+Definition provided in the notebook.
+
+### 3. How to run this tutorial (without TLS and locally as a simulation):
+
+0. Pre-requisites:
+
+    Activate virtual environment (Python - 3.8.10) and install below packages.
+
+```
+pip install openfl==1.3 protobuf==3.19.4 --no-cache-dir
+```
+
+1. Run director:
+
+```sh
+cd director_folder
+./start_director.sh
+```
+
+2. Run envoy:
+
+Step 1: Activate virtual environment and install packages from the requirements.txt
+```
+cd envoy_folder
+pip install -r requirements.txt --no-cache-dir
+```
+Step 2: start the envoy
+```sh
+./start_envoy.sh env_instance_1 envoy_config_1.yaml
```

Comment:
General tip: If `env_instance_x` is a name, name it as `Bob`, `Alice` etc. So that the reader understands it is not a flag or an argument that affects the execution. 

We've not been doing this on older examples (even I didn't for that matter). But I will make that change in my previous example.

Reply:
Would it be better to enclose `env_instance_x` in double quotes `"env_instance_x"` to let user know that it's a user defined string passed as a parameter? A generic naming like  `Bob, Alice` might take it a bit out of program's intent and probably naming convention for envoy instance. Let me know your thoughts on this.